modules/file-rotation/file-rotation.h

```diff
@@ -25,11 +25,56 @@
 #define FILE_ROTATION_H_INCLUDED
 
 #include "syslog-ng.h"
+#include "driver.h"
+#include "signal-slot-connector/signal-slot-connector.h"
 
 typedef struct _FileRotationPlugin FileRotationPlugin;
+typedef struct _LogDriver LogDriver;
+typedef struct _FileRotationResponseData FileRotationResponseData;
+typedef enum
+{
+  FILE_ROTATION_INTERVAL,
+  FILE_ROTATION_SIZE,
+} FileRotationOption;
+
+typedef enum
+{
+  FILE_ROTATION_SUCCESS,
+  FILE_ROTATION_FAILURE,
+} FileRotationResult;
+
+struct _FileFlushSignalData
+{
+  GString *filename;
+  gchar *interval;
+  gsize size;
```

Comment:
filename can be useful, but we don't need interval and size here.

Reply:
Removed